Package.swift

```diff
@@ -11,6 +11,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/ReactiveX/RxSwift", from: "6.5.0"),
+        .package(url: "https://github.com/mattgallagher/CwlPreconditionTesting.git", from: "2.2.2"), // for testTarget only
```

Comment:
Need add `CwlPreconditionTesting` to RIBs.podspec too.

`
  s.test_spec 'Tests' do |test_spec|
    test_spec.source_files = 'RIBsTests/**/*.swift'
    test_spec.dependency 'CwlPreconditionTesting'
  end
`